### PR TITLE
Add endpoint for updating user roles

### PR DESCRIPTION
**Pull request by johnpcooke94**
This PR will add an endpoint at api/user/change-role to allow functionality to update user roles from the admin panel on the front end. There will be a corresponding PR on the front end to add this functionality to the webapp.

### CODE REVIEW

**Review by Cliftonz — DISMISSED** [review]
lgtm

File: CSLabs.Api/Controllers/UserController.cs [hunk 1]
```diff
@@ -137,5 +137,26 @@ public async Task<IActionResult> ChangePassword(ChangePasswordRequest request)
 
             return Ok();
         }
+
+        [HttpPut("change-role")]
+        public async Task<IActionResult> ChangeRole(ChangeRoleRequest[] request)
```
Comment (jasekiw) [inline CSLabs.Api/Controllers/UserController.cs]: We typically standardize on the List concrete class for request data than arrays.
```suggestion
        public async Task<IActionResult> ChangeRole(List<ChangeRoleRequest> request)
```

File: CSLabs.Api/Controllers/UserController.cs [hunk 2]
```diff
@@ -137,5 +137,26 @@ public async Task<IActionResult> ChangePassword(ChangePasswordRequest request)
...
+            {
+                return Unauthorized();
+            }
+            
+            foreach (var requestItem in request)
+            {
+                var user = await DatabaseContext
+                    .Users
+                    .Where(u => u.Id == requestItem.UserId)
+                    .FirstOrDefaultAsync();
```
Comment (jasekiw) [inline CSLabs.Api/Controllers/UserController.cs]:
```suggestion
                var user = await DatabaseContext.Users.FirstOrDefaultAsync(u => u.Id == requestItem.UserId);
```

**Review by jasekiw — CHANGES_REQUESTED** [review]

File: CSLabs.Api/Controllers/UserController.cs (hunk 1 above)
Comment (johnpcooke94) [reply CSLabs.Api/Controllers/UserController.cs]: Pushed a commit for this

File: CSLabs.Api/Controllers/UserController.cs (hunk 2 above)
Comment (johnpcooke94) [reply CSLabs.Api/Controllers/UserController.cs]: Pushed a commit for this

**Review by jasekiw — APPROVED** [review]

**Pull request merged**